x-pack/plugins/cases/public/components/all_cases/all_cases_list.tsx

```diff
@@ -160,16 +155,15 @@ export const AllCasesList = React.memo<AllCasesListProps>(
 
     const onFilterChangedCallback = useCallback(
       (newFilterOptions: Partial<FilterOptions>) => {
+        // FIXME: how should this work with multiple options?
```

Comment:
This functionality should be removed as with the upcoming work users can select/deselect their columns as they like. We should not set the sorting field based on the filters anymore.

Reply:
removed! 🗑️